pkg/scheduler/api/devices/nvidia/vgpu/device_info.go

```diff
@@ -182,20 +185,30 @@ func (gs *GPUDevices) Release(kubeClient kubernetes.Interface, pod *v1.Pod) erro
 func (gs *GPUDevices) FilterNode(pod *v1.Pod) (int, string, error) {
 	if VGPUEnable {
 		klog.V(5).Infoln("4pdvgpu DeviceSharing starts filtering pods", pod.Name)
-		fit, _, err := checkNodeGPUSharingPredicate(pod, gs, true)
+		fit, _, score, err := checkNodeGPUSharingPredicate(pod, gs, true)
 		if err != nil || !fit {
 			klog.Errorln("deviceSharing err=", err.Error())
 			return devices.Unschedulable, fmt.Sprintf("4pdvgpuDeviceSharing %s", err.Error()), err
 		}
+		gs.ScoreMap[pod.Name] = score
 		klog.V(5).Infoln("4pdvgpu DeviceSharing successfully filters pods")
 	}
 	return devices.Success, "", nil
 }
 
+func (gs *GPUDevices) ScoreNode(pod *v1.Pod) float64 {
+	/* TODO: we need a base score to be campatable with preemption, it means a node without evicting a task has
+	a higher score than those needs to evict a task */
+
+	// Use cached stored in filter state in order to avoid recalculating.
+	klog.V(5).Infoln("pod=", pod.Name, "scoreMap=", gs.ScoreMap[pod.Name])
```

Comment:
Please use klog.InfoS and ErrorS format, and then `=` is not needed. And please modify other places too.

Reply:
ok